addon/components/plaid-link.js

```diff
@@ -1,15 +1,14 @@
-/* global Plaid:false */
-// Sets Plaid as a global read-only variable for eslint
-
 import Component from '@ember/component';
 import layout from '../templates/components/plaid-link';
+import { inject as service } from '@ember/service';
 
-const OPTIONS = ['clientName', 'env', 'key', 'product', 'webhook', 'token'];
+const OPTIONS = ['clientName', 'env', 'key', 'product', 'webhook', 'token', 'apiVersion'];
 const DEFAULT_LABEL = 'Link Bank Account'; // Displayed on button if no block is passed to component
 
 export default Component.extend({
   layout,
   tagName: 'button',
+  plaid: service(),
```

Comment:
I have mixed feelings about calling this service straight `plaid`.

Reply:
Same. I was thinking maybe of naming it plaidLinkerProxy. What do you think?